apps/envConstants.js

```diff
@@ -45,5 +45,7 @@ module.exports = {
   BROWSER: process.env.BROWSER,
   // If set, will turn on react hot loader and run the webpack dev server
   HOT: !!process.env.HOT,
-  LEVEL_TYPE: process.env.LEVEL_TYPE
+  LEVEL_TYPE: process.env.LEVEL_TYPE,
+  // Include static assets when serving storybook locally
+  STORYBOOK_STATIC_ASSETS: process.env.STORYBOOK_STATIC_ASSETS,
```

Comment:
Very new to this - would you mind explaining when/how `process.env.STORYBOOK_STATIC_ASSETS` is assigned?

Reply:
mmm yeah, when we run `npm run storybook`, I prefix starting storybook by setting an env variable `STORYBOOK_STATIC_ASSETS=1`.

Then, you can access your environment variables via `process.env` in Node:

https://nodejs.dev/en/api/v18/process/#processenv

Reply:
Thanks!